Source/Core/Core/HW/GCMemcard/GCMemcard.cpp

```diff
@@ -439,6 +439,15 @@ std::string GCMemcard::DEntry_BIFlags(u8 index) const
   return flags;
 }
 
+bool GCMemcard::DEntry_IsPingPong(u8 index) const
+{
+  if (!m_valid || index >= DIRLEN)
+    return false;
+
+  const int flags = GetActiveDirectory().m_dir_entries[index].m_banner_and_icon_flags;
+  return (flags & 0b0000100) != 0;
```

Comment:
You should probably make sure to use 8 total digits here if you stick to leading zeroes.

Reply:
Ah, didn't even notice that.